releasenotes/README.md

```diff
@@ -0,0 +1,93 @@
+# Istio Release Notes
+
+This directory contains the release notes, upgrade notes, and security notes for Istio.
+Notes should be created as part of the pull request for any user facing changes. Before
+a release, the release notes utility will be run in order to generate a release notes file
+which will be reviewed by the release managers and documentation team.
+
+## Adding a Release Note
+
+To create a release note, create a new file in the [./notes](./notes) directory based on
+the [template](./template.yaml). The filename doesn't matter as long as it ends with a `.yaml`
+extension and matches the format specified in the template. However, please make names descriptive.
+
+```yaml
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+
+# issue is a list of GitHub issues resolved in this note.
+issue:
+  - https://github.com/istio/istio/issues/23622
+  - https://github.com/istio/istio/issues/23623
+  - https://github.com/istio/istio/issues/23624
+releaseNotes: |
+*Fixed* an issue preventing the operator from recreating watched resources if they are deleted
+
+upgradeNotes:
+  - title: Change the readiness port of gateways
+    content: |
+      If you are using the 15020 port to check the health of your Istio ingress gateway with your Kubernetes network load balancer, change the port from 15020 to 15021.
+
+securityNotes: |
+__[CVE-2020-15104](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-15104)__:
+When validating TLS certificates, Envoy incorrectly allows a wildcard DNS Subject Alternative Name to apply to multiple subdomains. For example, with a SAN of `*.example.com`, Envoy incorrectly allows `nested.subdomain.example.com`, when it should only allow `subdomain.example.com`.
+    - CVSS Score: 6.6 [AV:N/AC:H/PR:H/UI:N/S:C/C:H/I:L/A:N/E:F/RL:O/RC:C](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:H/PR:H/UI:N/S:C/C:H/I:L/A:N/E:F/RL:O/RC:C&version=3.1)
+```
+
+Some release notes may affect multiple types of notes. For those, please fill in all respective areas. For notes that only affect one or two areas, please fill in only those sections. Sections that don't have content can be omitted.
```

Comment:
You mention omitting the sections, but the template has `None`. Will both create the same result?

Reply:
Kubernetes uses "" and `none` interchangeably for their release notes. I was originally going to follow that model, but I think it just adds extra complexity.